### PR TITLE
cfdg: new formula

### PR DESCRIPTION
**Pull request by kn1kn1**
cfdg is a command-line tool for Context Free Art.
http://www.contextfreeart.org/

### CODE REVIEW

File: Library/Formula/cfdg.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Cfdg < Formula
+  homepage 'http://www.contextfreeart.org/'
```
Comment (mikemcquaid) [inline Library/Formula/cfdg.rb]: Use double-quotes everywhere.

File: Library/Formula/cfdg.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  url 'http://www.contextfreeart.org/download/ContextFreeSource3.0.8.tgz'
+  sha1 'bbd90c74fd0e66cb071c342da52ad016e023da2b'
+
+  depends_on 'bison' => :build
+  depends_on 'libpng'
+
+  def install
+    system 'make'
+    bin.install 'cfdg'
```
Comment (mikemcquaid) [inline Library/Formula/cfdg.rb]: Is there no `make install` we can use?

File: Library/Formula/cfdg.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    rule 0.03 {
+      SQUARE []
+      SPIKE [r 60]
+      SPIKE [r -60]
+      SPIKE [y 0.95 s 0.97]
+    }
+    EOS
+    system 'cfdg', '-s', '700', 'test.cfdg', 'test.cfdg.png'
+  end
```
Comment (mikemcquaid) [inline Library/Formula/cfdg.rb]: Nice test :+1:

File: Library/Formula/cfdg.rb (hunk 2 above)
Comment (kn1kn1) [inline Library/Formula/cfdg.rb]: I investigated that before writing this formula, but there's no rule to install in Makefile.

**Comment by mikemcquaid** [comment]
Fails to build on 10.8: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19378/version=mountain_lion/console

**Comment by kn1kn1** [comment]
I guess there's something wrong with linking bison header files, but I do not have any 10.8 environment to hack for now...

**Comment by kn1kn1** [comment]
I would like to publish this formula as a tapped one. thanks for many comments.

**Pull request closed without merging**